docs/mobile/deployments.md

```diff
@@ -0,0 +1,27 @@
+## Mobile Playbook: Deployments
+
+#### Automate build and deployment with Fastlane
+
+#### Configuring apps to support multiple environments
+
+#### Certificates and provisioning profiles
+
+#### Continuous Integration
+Options:
+* Bitrise.io,
+* Jenkins + Private Build Server,
+* Microsoft VSTS + MacInCloud build agents)
+
```

Comment:
Travis and CircleCI are options, but the pricing for Mac build servers are somewhat overpriced. CircleCI is $249/mo for the minimum plan. Travis used to have a price listed but they have now removed it from public pricing page, would need to ask.

I feel in the hosted mobile CI startup space Bitrise is a clear winner, because their workflows are very much optimised for common mobile app build tasks. Though any machine with ssh access and ability to run Jenkins and Fastlane will do in practice.